daemon/src/model.rs

```diff
@@ -655,4 +655,16 @@ mod tests {
 
         assert_eq!(ts_b.seconds() - ts_a.seconds(), -36000);
     }
+
+    #[test]
```

Comment:
nice! do we have a rule of thumb when we should move things into a test module? I'm happy for it to stay as it is, as it seems to be a trivial test that won't bloat lib size or compile time, asking just out of curiosity.

Reply:
> that won't bloat lib size or compile time

That is why this is behind `cfg(test)` (line 548)! `cfd` is conditional compilation in Rust so non of this gets compiled unless you want to run the tests.

Reply:
oops, missed that one! 😅